.github/workflows/quality.yml

```diff
@@ -2,7 +2,7 @@ name: quality
 
 on: 
   push:
-    branches: [ $default-branch ]
+    branches: [master]
```

Comment:
i think it's => 
  branches: 
    - 'master'
 
i saw some examples that use this format => branches: [master] so i don't know the difference, but try it.

Reply:
If everything is fine just merge it

Reply:
for me it's all right, all the pushes that we'll do it'll be into the branch master.
so for every push event it'll trigger this workflow and the same with the pull_request.
so i believe that's all right.